### PR TITLE
@bentley/imodeljs-native 2.19.2

### PR DESCRIPTION
**Pull request by pmconne**

### CODE REVIEW

**Comment by pmconne** [comment]
@kabentley 3 imodeljs-backend tests are failing, looks like due to federation guid changes.
I think I'm supposed to make it expected this property **not** to be undefined but please review and confirm.
```
  1) Catalog
       should import from catalog:
     AssertionError: expected 'ff1e9a7f-5c33-44ee-bf38-b4e1ac3852cc' to equal undefined
      at Context.<anonymous> (src\test\standalone\Catalog.test.ts:736:16)

  2) iModel
       should update FederationGuid:
     AssertionError: expected '41f31dd0-a8c1-448f-b8fb-520cf16fc7e5' to equal undefined
      at Context.<anonymous> (src\test\standalone\IModel.test.ts:2435:12)
      at processImmediate (internal/timers.js:461:21)

  3) iModel
       should support partial update:
     AssertionError: expected 'd83b4665-5d3b-4640-a856-7094fe041965' to equal undefined
      at Context.<anonymous> (src\test\standalone\IModel.test.ts:2518:12)
      at processImmediate (internal/timers.js:461:21)
```

**Review by bsteinbk — APPROVED** [review]

**Comment by pmconne** [comment]
@kabentley this imodeljs-backend integration test is also failing.
```
  1) BriefcaseManager (#integration)
       should open iModels of specific versions from the Hub:
     AssertionError: expected undefined to equal 4
      at Context.<anonymous> (src/test/integration/BriefcaseManager.test.ts:139:12)
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (internal/process/task_queues.js:95:5)
```

**Review by kabentley — APPROVED** [review]

**Comment by pmconne** [comment]
@Mergifyio update

File: core/backend/src/IpcHost.ts [hunk 1]
```diff
@@ -214,16 +215,15 @@ class IpcAppHandler extends IpcHandler implements IpcAppFunctions {
     return IModelDb.findByKey(key).nativeDb.getUndoString();
   }
 
-  public async pullAndMergeChanges(key: string, version?: IModelVersionProps): Promise<string> {
+  public async pullAndMergeChanges(key: string, version?: IModelVersionProps): Promise<ChangesetIndexAndId> {
```
Comment (calebmshafer) [inline core/backend/src/IpcHost.ts:218]: @kabentley the change in return type here may be worth a mention in the NextVersion.md so people know how to respond to the change.

**Comment by pmconne** [comment]
Everything passed except extract-api for ecschema-metadata on windows, which is known unrelated issue.

**Pull request merged**